core/ui/src/main/res/values/strings.xml

```diff
@@ -205,4 +205,7 @@
     <string name="tree">Tree</string>
     <string name="played_progress">Played progress</string>
     <string name="media_view_mode">Media view mode</string>
+    <string name="left_bottom_corner">Left bottom corner</string>
+    <string name="right_bottom_corner">Right bottom corner</string>
+    <string name="player_control_buttons_position">Player control buttons position</string>
```

Comment:
How about changing this to something like` Control buttons alignment` and options should be left or right